rack-box/scripts/install.sh

```diff
@@ -70,6 +72,16 @@ cp /opt/fuseki/fuseki.service /etc/systemd/system/fuseki.service
 systemctl enable fuseki
 systemctl start fuseki
 
+# Set up and start RACK UI service
+
+cd /home/"${USER}"/RACK/rack-ui
+python3 -m pip install -r ./requirements.txt
+adduser --system --group --no-create-home --disabled-password rackui
+mkdir /etc/rackui
+chown rackui.rackui /etc/rackui
+envsubst < rackui.service > /etc/systemd/system/rackui.service
+systemctl enable rackui
+
```

Comment:
How are you starting the rack ui if it needs to be started?  Do you need to add `systemctl start rackui` at line 84 or do you want users to start it manually?